src/data/resolvers/stakes.ts

```diff
@@ -0,0 +1,61 @@
+import { Resolvers } from '@apollo/client';
+import { ClientType, Extension } from '@colony/colony-js';
+import { bigNumberify } from 'ethers/utils';
+
+import { Context } from '~context/index';
+import { MotionVote } from '~utils/colonyMotions';
+
+export const stakesResolvers = ({
+  colonyManager,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async stakeAmountsForMotion(
+      _,
+      { colonyAddress, userAddress, motionId, isObjectionStake },
+    ) {
+      try {
+        const supportedSide = isObjectionStake
+          ? MotionVote.NAY
+          : MotionVote.YAY;
+        const colonyClient = await colonyManager.getClient(
+          ClientType.ColonyClient,
+          colonyAddress,
+        );
+        const tokenDecimals = await colonyClient.tokenClient.decimals();
+        const votingReputationClient = await colonyClient.getExtensionClient(
+          Extension.VotingReputation,
+        );
+        const { stakes, skillRep } = await votingReputationClient.getMotion(
+          motionId,
+        );
+        const userStake = await votingReputationClient.getStake(
+          motionId,
+          userAddress,
+          supportedSide,
+        );
+        // @NOTE There's no prettier compatible solution to this :(
+        // eslint-disable-next-line max-len
+        const totalStakeFraction = await votingReputationClient.getTotalStakeFraction();
+        const requiredStake = skillRep
+          .mul(totalStakeFraction)
+          .div(bigNumberify(10).pow(tokenDecimals * 2))
```

Comment:
This exact logic is being used in three places:
- here, in your resolver
- in the actions tranformer: https://github.com/JoinColony/colonyDapp/blob/9a5afefb958163bf822bfec3c47fa2a013d1ef8e/src/modules/dashboard/transformers.ts#L113-L115
- in the events util: https://github.com/JoinColony/colonyDapp/blob/9a5afefb958163bf822bfec3c47fa2a013d1ef8e/src/utils/events/index.ts#L510-L512

Please refactor it into a util. Something like:
```
getMotionRequiredStake(skillRep, totalStakeFraction, tokenDecimals);
```